src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+$bodyFont: 'Roboto';
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  height: 406px;
+  width: 200px;
 }
```

Comment:
Setting a fixed width and height on the body tag is not a common practice and can lead to responsiveness issues. Instead, consider using min-width or min-height if necessary, or controlling the size of content through container elements.